### ISSUE TITLE
Developer Talks

### ISSUE THREAD

**Issue by ryandonsullivan** [norcross/ama#5]
Which types of talks would you like to see more of at developer conferences? I personally feel like there's a lot of flash and fancy that goes on a lot of the time, but that can make it tough for practicality, which might be even more important than the "this is possible maybe" types of talks. What do you think?

**Comment by norcross** [comment]
I believe it's difficult to give a developer focused talk without there being certain things in place, namely the crowd being aware of general skill level and required background information to make the talk valuable. For me, seeing developers give more "high level" talks has more value, since I can take the points and notes then do the research on my own afterwards. Watching someone code for 20 minutes and talk for 20 minutes doesn't really help anyone.

Also, there's a lot of developers (myself included) that struggle with the best way to _explain_ code to people in plain English. It's easy to just point to the code and say **do this** but a lot harder to get into the _why_ and _how_ of it all.

I believe the perfect mix for a conference is having talks that touch on new tools / methods / frameworks without a lot of heavy details (but resources to find them) combined with workshops that are longer in time that allow for deep diving into these things, with smaller crowds, where the presenter can actually interact with the people there and go into specifics on something.

**Comment by ryandonsullivan** [comment]
I hear all of that. Awesome answer. Thanks man :)

**Comment by norcross** [comment]
anytime hoss
